NODE_ENV prefix throws error in Windows

```
> react-toolbox@0.11.5 build E:\GitHub\react-toolbox
> NODE_ENV=production npm run babel && npm run sass

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v4.2.2
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! react-toolbox@0.11.5 build: `NODE_ENV=production npm run babel && npm run sass`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-toolbox@0.11.5 build script 'NODE_ENV=production npm run babel && npm run sass'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-toolbox package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=production npm run babel && npm run sass
npm ERR! You can get their info via:
npm ERR!     npm owner ls react-toolbox
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     E:\GitHub\react-toolbox\npm-debug.log
```

Seems prefixing the environment variable when invoking an NPM script doesn't work in Windows when checking out the project source and running `npm install` (**Note:** Getting from NPM as a dependency still works fine)

--- a/docs/package.json
+++ b/docs/package.json
@@ -47,7 +47,7 @@
     "react-transform-catch-errors": "^1.0.0",
     "react-transform-hmr": "^1.0.1",
     "redbox-react": "^1.1.1",
-    "sass-loader": "^3.1.1",
+    "sass-loader": "^3.0.0",
     "style-loader": "^0.13.0",
     "toolbox-loader": "0.0.2",
     "transfer-webpack-plugin": "^0.1.4",

--- a/docs/package.json
+++ b/docs/package.json
@@ -22,7 +22,6 @@
     "babel-eslint": "^4.1.3",
     "babel-loader": "^5.3.2",
     "babel-plugin-react-transform": "^1.1.1",
-    "better-npm-run": "0.0.3",
     "cross-env": "^1.0.4",
     "css-loader": "^0.21.0",
     "express": "^4.13.3",

--- a/docs/package.json
+++ b/docs/package.json
@@ -31,7 +31,7 @@
     "babel-eslint": "^4.1.3",
     "babel-loader": "^5.3.2",
     "babel-plugin-react-transform": "^1.1.1",
-    "better-npm-run": "^0.0.3",
+    "better-npm-run": "0.0.3",
     "css-loader": "^0.21.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.8.2",

--- a/docs/package.json
+++ b/docs/package.json
@@ -5,8 +5,17 @@
   "author": "React Toolbox Team (http://github.com/react-toolbox)",
   "scripts": {
     "start": "node ./server",
-    "build": "NODE_ENV=production UV_THREADPOOL_SIZE=100 webpack --config ./webpack.config.production",
+    "build": "better-npm-run build",
     "deploy": "gh-pages -d build"
+  },
+  "betterScripts": {
+    "build": {
+      "command": "webpack --config ./webpack.config.production --colors --profile --progress",
+      "env": {
+        "NODE_ENV": "production",
+        "UV_THREADPOOL_SIZE": 100
+      }
+    }
   },
   "dependencies": {
     "codemirror": "^5.8.0",
@@ -22,6 +31,7 @@
     "babel-eslint": "^4.1.3",
     "babel-loader": "^5.3.2",
     "babel-plugin-react-transform": "^1.1.1",
+    "better-npm-run": "^0.0.3",
     "css-loader": "^0.21.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.8.2",
@@ -37,7 +47,7 @@
     "react-transform-catch-errors": "^1.0.0",
     "react-transform-hmr": "^1.0.1",
     "redbox-react": "^1.1.1",
-    "sass-loader": "^3.0.0",
+    "sass-loader": "^3.1.1",
     "style-loader": "^0.13.0",
     "toolbox-loader": "0.0.2",
     "transfer-webpack-plugin": "^0.1.4",

--- a/docs/package.json
+++ b/docs/package.json
@@ -5,17 +5,8 @@
   "author": "React Toolbox Team (http://github.com/react-toolbox)",
   "scripts": {
     "start": "node ./server",
-    "build": "better-npm-run build",
+    "build": "cross-env NODE_ENV=production UV_THREADPOOL_SIZE=100 webpack --config ./webpack.config.production --colors --profile --progress",
     "deploy": "gh-pages -d build"
-  },
-  "betterScripts": {
-    "build": {
-      "command": "webpack --config ./webpack.config.production --colors --profile --progress",
-      "env": {
-        "NODE_ENV": "production",
-        "UV_THREADPOOL_SIZE": 100
-      }
-    }
   },
   "dependencies": {
     "codemirror": "^5.8.0",
@@ -32,6 +23,7 @@
     "babel-loader": "^5.3.2",
     "babel-plugin-react-transform": "^1.1.1",
     "better-npm-run": "0.0.3",
+    "cross-env": "^1.0.4",
     "css-loader": "^0.21.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.8.2",

--- a/package.json
+++ b/package.json
@@ -21,12 +21,32 @@
     "lint": "eslint ./ --ext .js,.jsx",
     "babel": "babel ./components --out-dir ./lib",
     "sass": "cpx './components/**/*.scss' ./lib",
-    "build": "NODE_ENV=production npm run babel && npm run sass",
+    "build": "better-npm-run build",
     "clean": "rimraf ./lib",
     "prebuild": "npm run clean",
     "prepublish": "npm run build",
-    "test": "NODE_ENV=test karma start",
-    "test:watch": "NODE_ENV=test karma start --no-single-run"
+    "test": "better-npm-run test",
+    "test:watch": "better-npm-run test:watch"
+  },
+  "betterScripts":{
+    "build": {
+      "command": "npm run babel && npm run sass",
+      "env": {
+        "NODE_ENV": "production"
+      }
+    },
+    "test": {
+      "command": "karma start",
+      "env": {
+        "NODE_ENV": "test"
+      }
+    },
+    "test:watch":{
+      "command": "karma start --no-single-run",
+      "env": {
+        "NODE_ENV": "test"
+      }
+    }
   },
   "bugs": {
     "url": "https://github.com/react-toolbox/react-toolbox/issues",
@@ -54,6 +74,7 @@
     "babel-loader": "^5.3.2",
     "babel-plugin-react-transform": "^1.1.1",
     "core-js": "^1.2.6",
+    "better-npm-run": "^0.0.3",
     "cpx": "^1.2.1",
     "css-loader": "^0.21.0",
     "eslint": "^1.7.3",
@@ -84,7 +105,7 @@
     "sass-loader": "^3.0.0",
     "sinon": "git://github.com/sinonjs/sinon.git#b672042043517b9f84e14ed0fb8265126168778a",
     "style-loader": "^0.13.0",
-    "toolbox-loader": "0.0.2",
+    "toolbox-loader": "0.0.3",
     "webpack": "^1.12.0",
     "webpack-dev-middleware": "^1.2.0",
     "webpack-hot-middleware": "^2.4.1"

--- a/package.json
+++ b/package.json
@@ -21,32 +21,12 @@
     "lint": "eslint ./ --ext .js,.jsx",
     "babel": "babel ./components --out-dir ./lib",
     "sass": "cpx './components/**/*.scss' ./lib",
-    "build": "better-npm-run build",
+    "build": "cross-env NODE_ENV=production npm run babel && npm run sass",
     "clean": "rimraf ./lib",
     "prebuild": "npm run clean",
     "prepublish": "npm run build",
-    "test": "better-npm-run test",
-    "test:watch": "better-npm-run test:watch"
-  },
-  "betterScripts":{
-    "build": {
-      "command": "npm run babel && npm run sass",
-      "env": {
-        "NODE_ENV": "production"
-      }
-    },
-    "test": {
-      "command": "karma start",
-      "env": {
-        "NODE_ENV": "test"
-      }
-    },
-    "test:watch":{
-      "command": "karma start --no-single-run",
-      "env": {
-        "NODE_ENV": "test"
-      }
-    }
+    "test": "cross-env NODE_ENV=test karma start",
+    "test:watch": "cross-env NODE_ENV=test karma start --no-single-run"
   },
   "bugs": {
     "url": "https://github.com/react-toolbox/react-toolbox/issues",
@@ -74,7 +54,7 @@
     "babel-loader": "^5.3.2",
     "babel-plugin-react-transform": "^1.1.1",
     "core-js": "^1.2.6",
-    "better-npm-run": "^0.0.3",
+    "cross-env": "^1.0.4",
     "cpx": "^1.2.1",
     "css-loader": "^0.21.0",
     "eslint": "^1.7.3",

--- a/package.json
+++ b/package.json
@@ -105,7 +105,7 @@
     "sass-loader": "^3.0.0",
     "sinon": "git://github.com/sinonjs/sinon.git#b672042043517b9f84e14ed0fb8265126168778a",
     "style-loader": "^0.13.0",
-    "toolbox-loader": "0.0.3",
+    "toolbox-loader": "0.0.2",
     "webpack": "^1.12.0",
     "webpack-dev-middleware": "^1.2.0",
     "webpack-hot-middleware": "^2.4.1"

